src/Featurizers/Components/DocumentStatisticsEstimator.h

```diff
@@ -128,8 +119,10 @@ class DocumentStatisticsTrainingOnlyPolicy {
     //       rather than asci characters.
     using InputType                         = std::string;
     using InputTypeConstIterator            = std::string::const_iterator;
-    using StringDecorator                   = std::function<std::string (InputTypeConstIterator begin, InputTypeConstIterator end)>;
-    using IndexMap                          = DocumentStatisticsAnnotationData::IndexMap;
+    using StringDecorator                   = std::function<std::string (std::string)>;
+    using IndexMap                          = std::unordered_map<std::string, std::uint32_t>;
+    using FrequencyMap                      = IndexMap;
+    using RegexT                            = std::regex;
```

Comment:
>using RegexT                            = std::regex; [](start = 4, length = 53)

It doesn't look like this is being used.

Reply:
it is used in fit_impl